### PR TITLE
Remove min-height and add blur to backdrop in modal

### PR DESCRIPTION
**Pull request by isacoder**
### Requirement

1.- Some designs have less than 300px height and current implementation doesn't allow to implement those.
2.- @atomworks mention that it would be great that the backdrop has blur.

### Implementation

Reviewed a couple of options fr the min-height with @atomworks and we decided just removing it will be better for now.

### Screenshots 

New Empty Modal
Screen Shot 2021-04-05 at 12 59 12

Previous templates, since content was bigger than 300px won't show any  difference
Screen Shot 2021-04-05 at 15 19 17

### CODE REVIEW

File: src/Alerts/atom/Modal.tsx [hunk 1]
```diff
@@ -16,7 +16,10 @@ const Container = styled.div`
   align-items: center;
   justify-content: center;
   background-color: hsl(0, 0%, 0%, 0.2);
+  -webkit-backdrop-filter: blur(5px);
+  backdrop-filter: blur(5px);
```
Comment (atomworks) [inline src/Alerts/atom/Modal.tsx:20]: This is way easier than the method we had to use before! I'm glad this has better browser support at last! 🥳

File: src/Alerts/atom/Modal.tsx [hunk 2]
```diff
@@ -16,7 +16,10 @@ const Container = styled.div`
   align-items: center;
   justify-content: center;
   background-color: hsl(0, 0%, 0%, 0.2);
```
Comment (atomworks) [inline src/Alerts/atom/Modal.tsx:18]: I wasn't sure about this when we added the initial version but this color is incorrect. It's more of a grey with a blueish hue. Zeplin says: `hsla(202, 33%, 95%, 0.51)`

**Review by atomworks — CHANGES_REQUESTED** [review]
Looks great. One very quick semi-related request though if you can! Just a color tweak.

File: src/Alerts/atom/Modal.tsx (hunk 2 above)
Comment (isacoder) [reply src/Alerts/atom/Modal.tsx]: Thank you for making this correction.
I have updated.

**Review by atomworks — APPROVED** [review]
Looks great, thanks!

**Pull request merged**